vitest.d.ts

```diff
@@ -0,0 +1,5 @@
+import "@testing-library/jest-dom";
+
+declare module "vitest" {
+  interface Assertion<T = any> extends jest.Matchers<void, T> {}
```

Comment:
Handle all these lint and type errors.
If the `any` type is definitely necessary then add this line to ignore the error:
```
// eslint-disable-next-line @typescript-eslint/no-explicit-any
```